vizro-core/docs/pages/user-guides/deploy.md

```diff
@@ -0,0 +1,195 @@
+
+## Ploomber Cloud
+
+[Ploomber Cloud](https://ploomber.io/) is a platform specifically built to deploy data visualization apps built using frameworks such as Vizro. Its free tier offers an easy deployment by drag and drop, the [Ploomber Cloud CLI](https://docs.cloud.ploomber.io/en/latest/user-guide/cli.html), or `git push`. Paid plans include features such as a custom domain, enterprise-grade authentication, user analytics and more powerful computing resources.
+
+The [Ploomber Cloud documentation](https://docs.cloud.ploomber.io/en/latest/apps/vizro.html) contains detailed instructions on how to deploy Vizro on Ploomber Cloud. In short, the process is as follows:
+
+1. [Sign up for a Ploomber Cloud account](https://platform.ploomber.io/).
+1. Modify the last line of your `app.py` to [expose the Vizro `app` object](#overview) as `app = Vizro().build(dashboard)`.
+1. Create a `requirements.txt` file to give your Python package dependencies. This should include `vizro` and `gunicorn`. See our [section on dependencies](#dependencies) for more information.
+1. Create a `Dockerfile` by copying the [example given in the Ploomber Cloud documentation](https://docs.cloud.ploomber.io/en/latest/apps/vizro.html#application-setup). See our [section on Dockerfiles](#dockerfile) for more information.
+1. Compress your `app.py`, `requirements.txt` and `Dockerfile` into a single zip file.
+1. Upload the zip file to Ploomber Cloud.
+
+<!-- TODO: https://platform.ploomber.io/register -->
+
+## Dash Enterprise
+
+Since a Vizro app is a Dash app under the hood, they can be deployed to [Dash Enterprise](https://plotly.com/dash/) and accessed in the same way as other Dash apps.
+
+Dash Enterprise helps to deploy and scale production-grade data apps and integrate them with IT infrastructure such as authentication and VPC services. Vizro users may find it suitable for deployment, rapid development environments, and authentication.
+
+Vizro is compatible with the following functionality within Dash Enterprise:
+
+- [App Portal](https://dash.plotly.com/dash-enterprise/portal?de-version=5.5)
+- [App Manager](https://plotly.com/dash/app-manager/)
+- [Dash App Workspaces](https://plotly.com/dash/workspaces/)
+- [App logs and viewer statistics](https://dash.plotly.com/dash-enterprise/logs?de-version=5.5)
```

Comment:
```suggestion
- [App logs and viewer statistics](https://dash.plotly.com/dash-enterprise/logs)
```
we should probably take out version so it stays current and goes to the newest version first time you visit